src/ui/api/statistic.go

```diff
@@ -64,17 +64,22 @@ func (s *StatisticAPI) Get() {
 	}
 
 	statistic[PubPC] = (int64)(len(pubProjs))
```

Comment:
This line should also be removed?

Reply:
It's the count of public projects, it's needed.